### DISCUSSION TITLE
WSL2 - kustomize: command not found

### DISCUSSION THREAD

**Discussion by thepaulmacca** [fluxcd/flux2#1896] [Q&A] [answered]
I'm following the docs here - https://fluxcd.io/docs/use-cases/azure/#flux-installation-for-azure-devops and when I get to this part:

![image](https://user-images.githubusercontent.com/41340540/135923139-fb2ee3d4-5f36-4a23-b6e3-169c03e41d3d.png)

I get a `kustomize: command not found` error when running `cd ./clusters/my-cluster/flux-system && kustomize create --autodetect`

I'm running Windows 10 with WSL2 (Ubuntu) installed. Can anyone help?

**Comment by thepaulmacca** [comment, marked as answer]
I've resolved this myself now

**Reply by khauser** [reply]
Glad that you've made it! But the world wants to know how?

**Reply by thepaulmacca** [reply]
I just had to install Kustomize in WSL:

```bash
curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash

sudo mv kustomize /usr/local/bin
```

**Reply by khauser** [reply]
Thank you very much
